docs/pages/es/project/CONTRIBUTING-gitflow.md

```diff
@@ -0,0 +1,95 @@
+# Trabajando por Fork
+Simplemente dirígete a la página de GitHub y haz clic en el botón "Fork". Es así de simple. Una vez que hayas hecho eso, puedes usar tu cliente git favorito para clonar tu repositorio o simplemente dirigirse directamente a la línea de comando:
+
+## Clona tu fork en tu máquina local
+```
+git clone git@github.com:USERNAME/FORKED-PROJECT.git
+```
+Mantener Tu Fork Actualizado
+Si bien este no es un paso necesario, si planeas hacer algo más que una pequeña solución rápida, querrás asegurarte de mantener tu fork actualizado rastreando el repositorio "upstream" original que bifurcaste. Para hacer esto, deberás agregar un "remote":
```

Comment:
Rename "original que bifurcaste" to "original al que hiciste fork" it's not recommended to transalte "fork" to "bifurcación". Also during the rest of the text you're using the word: fork. so keep it :+1: